craft_application/application.py

```diff
@@ -146,7 +150,7 @@ def _configure_services(self, build_for: str | None) -> None:
     @functools.cached_property
     def project(self) -> models.Project:
         """Get this application's Project metadata."""
-        project_file = (self._work_dir / f"{self.app.name}.yaml").resolve()
+        project_file = (pathlib.Path.cwd() / f"{self.app.name}.yaml").resolve()
```

Comment:
Maybe something like `pathlib.Path(self.app.name + ".yaml").resolve()` could be more readable?